src/main/scala/loamstream/apps/Main.scala

```diff
@@ -149,26 +149,24 @@ object Main extends Loggable {
       info(compilationResult.report)
       
       if(compilationResult.isSuccess && compilationResult.isValid) {
-        val executables = compilationResult.graphSource.iterator.map(thunk => LoamEngine.toExecutable(thunk()))
+        val executableOpt = compilationResult.contextOpt.map(_.graph).map(LoamEngine.toExecutable(_))
```

Comment:
This is the second time in the review I'm seeing:
```
compilationResult.contextOpt.map(_.graph)
```
Would it make sense to have that as a `lazy val` or `def` member of `compilationResults`? Perhaps not, just thinking DRY.

Reply:
Sure, something like `compilationResults.graphOpt` would be handy.